src/hooks/useSubscriptionPlan.ts

```diff
@@ -0,0 +1,28 @@
+import {useOnyx} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+function useSubscriptionPlan() {
+    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
+
+    if (!policies) {
+        return null;
+    }
+
+    const adminPolicies = Object.fromEntries(
```

Comment:
Do you think it should be memorized somehow? Does it update once?

Reply:
I mean to avoid extra updates.

Reply:
I wrapped it in `useMemo` according to @kubabutkiewicz 's comment